src/main.js

```diff
@@ -50,6 +50,26 @@ async function main() {
         }
     }
 
+    /**
+     * In development mode, we include the heapdump module if it exists.
+     * With this installed, `kill -USR2 <pid>` can be used to create a
+     * heapsnapshot file of the gateway's memory.
+     */
+    if (process.env.KA_ALLOW_HEAPDUMP || process.env.NODE_ENV === "dev") {
```

Comment:
I think it should be "development"!

Reply:
Sadly, no. RRS uses `dev` as seen on [line 45](https://github.com/Khan/react-render-server/blob/f20688735178ba05a2dfb6adea3437847e37e475/src/main.js#L45)